earthaccess/__init__.py

```diff
@@ -58,6 +59,9 @@
     "Store",
     # kerchunk
     "consolidate_metadata",
+    # virtualizarr
+    "open_virtual_dataset",
+    "open_virtual_mfdataset",
```

Comment:
We should probably add a `virtualizarr.open_virtual_mfdataset` upstream in virtualizarr. Though I would like to be more confident about the best way to parallelize reference generation first

https://github.com/zarr-developers/VirtualiZarr/issues/123

https://github.com/zarr-developers/VirtualiZarr/issues/7

Reply:
tracked in https://github.com/zarr-developers/VirtualiZarr/issues/345